pylint.sh

```diff
@@ -12,4 +12,4 @@ export PYTHONPATH=$PYTHONPATH:$WORKSPACE/gpkit/
 
 python $PYLINT --rcfile=$WORKSPACE/.pylintrc --disable=fixme $@ $WORKSPACE/gpkit/
 
-python $PYLINT --rcfile=$WORKSPACE/.pylintrc --disable=invalid-name,too-many-locals,redefined-outer-name,wrong-import-position $@ $WORKSPACE/docs/source/examples/*.py
+python $PYLINT --rcfile=$WORKSPACE/.pylintrc --disable=invalid-name,too-many-locals,redefined-outer-name,wrong-import-position,missing-docstring $@ $WORKSPACE/docs/source/examples/*.py
```

Comment:
I disagree with disabling this -- we're so close to having docstrings on all our methods, we should strive to do so.

There's a separate pylint threshold for allowing trivial methods shorter than a certain length to not have a dicstring.